integration/operator_test.go

```diff
@@ -0,0 +1,235 @@
+// +build k8srequired
+
+package integration
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cenk/backoff"
+	"github.com/giantswarm/certificatetpr"
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	v1 "k8s.io/client-go/pkg/api/v1"
+)
+
+const (
+	certOperatorValuesFile = "/tmp/cert-operator-install.yaml"
+	defaultDeadline        = 15
```

Comment:
Nitpick but defaultDeadline reads a bit strangely to me since this is a timeout.

Reply:
Sure, much better, done.